docs/specviz/notebook.rst

```diff
@@ -1,5 +1,19 @@
 ***********************************
-Using SpecViz in a Jupyter Notebook 
+Using Specviz in a Jupyter Notebook 
 ***********************************
 
-Words go here...
+Specviz is developed to fully support analyzing spectra within your existing `Jupyter notebooks <https://jupyter.org/>`_! To use Specviz, install jdaviz in your notebook's python environment and add a new cell wherever you would like to use Specviz.
+
+    >>> # Import specviz
+    >>> from jdaviz import SpecViz
+    >>> # Instantiate an instance of Specviz
+    >>> specviz = SpecViz()
+    >>> # Display Specviz
+    >>> specviz.app
```

Comment:
There's a CI failure due to this line - basically it's expecting no output but of course this should have an output. But there's no need to show in these docs so I'd say just do:
```suggestion
    >>> specviz.app   #doctest: +SKIP
```